src/components/textarea/textarea.spec.tsx

```diff
@@ -267,6 +267,32 @@ describe("Textarea", () => {
               mockedGuid
             );
           });
+
+          it("should only render one DOM element when labelHelp is also passed", () => {
```

Comment:
comment: I think the test below asserting against the text is all that we need here